psutil/_psutil_windows.c

```diff
@@ -1210,6 +1209,15 @@ psutil_users(PyObject *self, PyObject *args) {
     if (py_retlist == NULL)
         return NULL;
 
+    // If either or those APIs are missing, we are probably running
+    // in a Windows Nano Server container
+    if (WTSEnumerateSessions == NULL ||
+        WTSQuerySessionInformation == NULL ||
+        WTSFreeMemory == NULL) {
+        PyErr_SetString(PyExc_RuntimeError, "wtsapi32.dll cannot be found.");
+        goto error;
+    }
```

Comment:
I'm not sure I understand. Are these names available on Windows Nano? Does this function (`users()`) work at all?

Reply:
If Windows Nano does not support users at all then I think it makes more sense to just return an empty list. It's less disruptive than an error.

Reply:
I think `Py_BuildValue("[]");` should do it.